libs/api/domains/vehicles/src/lib/api-domains-vehicles.service.ts

```diff
@@ -151,10 +153,18 @@ export class VehiclesService {
               ? data.plates[0].reggroup
               : null
             : null,
+          reggroupName: data.plates
+            ? data.plates.length > 0
+              ? data.plates[0].reggroupname
+              : null
+            : null,
```

Comment:
```suggestion
          reggroupName: data.plates?.[0]?.reggroupname ?? null
```
án þess að vita nákvæmlega hvernig gögnin geta komið, en hægt að stytta og gera læsilegra